src/DIRAC/Core/Utilities/Graphs/QualityMapGraph.py

```diff
@@ -171,7 +171,7 @@ def draw(self):
         cb = ColorbarBase(
             cax, cmap=self.cmap, norm=self.norms, boundaries=self.cbBoundaries, values=self.cbValues, ticks=self.cbTicks
         )
-        cb.draw_all()
+        cb._draw_all()
```

Comment:
Shouldn't this have been fixed by https://github.com/DIRACGrid/DIRAC/pull/7210

Reply:
Sorry, I didn't check for such a pull request. Let me rebase and see if I still get the CI error.